### ISSUE TITLE
Python plot_sinogram_profiles confused by TOF

### ISSUE THREAD

**Issue by KrisThielemans** [UCL/STIR#1363]
data is now 4D, so the int sumaxis,select leads to a load of profiles.

@robbietuk can we modify this to pass tuples? Or alternatively "sumallotheraxes"

**Comment by robbietuk** [comment]
Yes. Let me take a look and get back to you. Behavior control is quite hard. It might be better to merge this with https://github.com/UCL/STIR/blob/master/examples/python/projdata_visualisation/ProjDataVisualisation.py

Some notes for myself:
This utility is hard.
- The idea is to plot a 1D line profile through (multiple) ProjData.
- This means each dimension of the data should be: a selected value or summed over all rows.
- 2D data: The dimension to plot across should be given (i.e. horizontal on plot) and vertical should be a value of the other ProjData dimension or a sum of it.
- In 3D The horizontal  component dimension should be provided and the vertical should be the given values of the other dimensions or a sum.
- This extends to 4D data.
